JSDemos/Demos/Toolbar/Multiline/Angular/app/app.component.html

```diff
@@ -0,0 +1,188 @@
+<div>
+  <dx-resizable
+    class="resizable-container"
+    [minWidth]="500"
+    [minHeight]="150"
+    [maxHeight]="370"
+    handles="right"
+  >
+    <dx-toolbar [multiline]="multiline">
+      <dxi-item
+        location="before"
+        locateInMenu="never"
+        widget="dxButton"
+        [options]="undoButtonOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="never"
+        widget="dxButton"
+        [options]="redoButtonOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        menuItemTemplate="menuSeparatorTemplate"
+      >
+        <div *dxTemplate>
+          <div class="separator"></div>
+        </div>
+      </dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        widget="dxDropDownButton"
+        [options]="fontSizeOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        widget="dxDropDownButton"
+        [style.line-height]="lineHeight"
+        [options]="lineHeightOptions"
+      ></dxi-item>
+
+      <dxi-item location="before" locateInMenu="never">
+        <div *dxTemplate>
+          <div class="separator"></div>
+        </div>
+      </dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        widget="dxSelectBox"
+        [options]="fontSelectorOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        menuItemTemplate="menuSeparatorTemplate"
+      >
+        <div *dxTemplate>
+          <div class="separator"></div>
+        </div>
+      </dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="never"
+        widget="dxButtonGroup"
+        [options]="fontStyleOptions"
+      ></dxi-item>
+
+      <dxi-item location="before" locateInMenu="never">
+        <div *dxTemplate>
+          <div class="separator"></div>
+        </div>
+      </dxi-item>
+
+      <dxi-item
```

Comment:
Let's try use the approach with templates instead of using options

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/files#diff-3df59d12600af6a791b2d6b1ee0dfa564c0c05e4dfbb76db4b63b78e32ce4586R91